localstack/services/awslambda/lambda_utils.py

```diff
@@ -262,7 +262,10 @@ def get_zip_bytes(function_code):
             s3_client.download_fileobj(function_code["S3Bucket"], function_code["S3Key"], bytes_io)
             zip_file_content = bytes_io.getvalue()
         except Exception as e:
-            raise ClientError("Unable to fetch Lambda archive from S3: %s" % e, 404)
+            s3_url = (
+                f's3://{function_code["S3Bucket"]}/{function_code.get("S3Key", "").lstrip("/")}'
```

Comment:
sure about that lstrip? :thinking: 
Think I recently did something similar with lambda layers where it then resolved to `s3.... //layer.zip` for example

Reply:
Yep, good catch! Thought about that too for a moment, will update 👍